### ISSUE TITLE
PDF export on save doesn't work when live preview is not enabled

### ISSUE THREAD

**Issue by marpala** [shd101wyy/markdown-preview-enhanced#804]
I can only export to PDF on save when the markdown preview tab is open.

**Comment by shd101wyy** [comment]
Yes you are right. Right now you need to have the preview open; otherwise the `export on save` will not work. 

I will support export on save without requiring preview to be opened in the future.

Thank you

**Comment by Champitoad** [comment]
Hi @shd101wyy ,

Just wanted to know if the feature was still planned to be added in the near future? I am writing a report with a custom LaTeX template in pandoc, thus the preview tab is not powerful enough in my usecase. Having the possibility to update the PDF when saving without the preview polluting the screen would be quite helpful!
